web/src/features/charts/tooltips/BreakdownChartTooltip.tsx

```diff
@@ -201,6 +203,12 @@ export function BreakdownChartTooltipContent({
               {__('tooltips.ofinstalled')}
               <br />
               <MetricRatio value={usage} total={(capacity ??= 0)} format={formatPower} />
+              {capacitySource && (
+                <small>
+                  {' '}
+                  ({__('country-panel.source')}: {capacitySource})
```

Comment:
Do you know if we have more than one source for a single capacity mode for any zone?

Reply:
I'm 99% sure we don't. In the future if we add sources for the annual view then there will probably be multiple sources for one generation type and hopefully this pattern will work

Reply:
Yeah otherwise we just add a intl list format and it will look all nice with multiple entries.